tests/sentry/event_manager/test_event_manager.py

```diff
@@ -285,6 +291,31 @@ def test_marks_as_unresolved_with_new_release(
 
         mock_send_activity_notifications_delay.assert_called_once_with(activity.id)
 
+    def test_has_pending_commit_resolution(self):
```

Comment:
Should you also have a test case for multiple commits where there is an unreleased one before a released one?

Reply:
Added